string name
---
float64 x
float64 y
bool task_available